### PR TITLE
giter8 0.6.8

### PR DESCRIPTION
**Pull request by justjoheinz**
update version to 0.6.8

### CODE REVIEW

File: Library/Formula/giter8.rb [hunk 1]
```diff
@@ -4,7 +4,7 @@ class Giter8 < Formula
   homepage "https://github.com/n8han/giter8"
   url "https://repo.typesafe.com/typesafe/ivy-releases/org.scala-sbt/sbt-launch/0.13.7/sbt-launch.jar"
```
Comment (mikemcquaid) [inline Library/Formula/giter8.rb]: Do we not need to update that?

**Comment by jacknagel** [comment]
Reading through this formula, we should really try to find an alternate way to package this software, as downloading the actual software at first run is counter to our normal policy.

**Comment by DomT4** [comment]
Any movement here?

**Comment by DomT4** [comment]
Closing in favour of #40358.

**Pull request closed without merging**